contracts/GenArt721CoreV3.sol

```diff
@@ -78,6 +79,12 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
     IAdminACLV0 public adminACLContract;
 
     struct Project {
+        uint24 invocations;
+        uint24 maxInvocations;
+        uint24 scriptCount;
+        uint64 completedTimestamp;
```

Comment:
nit: can we please make an inline comment clarifying that `64` bytes is plenty for the life of all of our existences and many generations (all of them) to come?

Reply:
570 billion years 😄 updated!

Reply:
Haha thank you.